python/cufinufft/cufinufft/_simple.py

```diff
@@ -0,0 +1,54 @@
+from cufinufft import Plan
+
+def nufft1d1(x, data, n_modes=None, out=None, eps=1e-6, isign=1):
```

Comment:
I'm super excited about these 6 interfaces, since they act on GPU pointers, right?  (no H2D/D2H transfer...) ?

Reply:
Yes the take PyCUDA arrays (so no pointers, but almost).